kiliautoml/models/_detectron2_semantic_segmentation.py

```diff
@@ -168,6 +185,9 @@ def train(
         kili_print(eval_res)
         kili_print(f"Evaluations results are available in {eval_dir}")
         kili_print("The logs and model are saved in ", cfg.OUTPUT_DIR)
+
+        if "segm" not in eval_res:
+            kili_print("Your Epoch number is probably too low.")
```

Comment:
`if "segm" not in eval_res:` means there is no segmentation result? In this case this should be printed:`"No prediction in the training evaluation: your Epoch number may be too low.")